suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx

```diff
@@ -22,12 +23,17 @@ import {
     AuthorizeDeviceStackRoutes,
     RootStackParamList,
     StackToStackCompositeNavigationProps,
+    useScrollView,
 } from '@suite-native/navigation';
 
 import { EnterPassphraseOnTrezorButton } from './EnterPassphraseOnTrezorButton';
 
 const FORM_CARD_PADDING = 12;
 
+// The scroll behaves differently for each platform, so we need to adjust the offset so make it always scroll to correct position.
```

Comment:
Please elaborate how the behavior is different that it needs different offset

Reply:
38ecda525e